boudicca.base/query-lib/src/main/kotlin/base/boudicca/query/evaluator/SimpleEvaluator.kt

```diff
@@ -8,26 +8,15 @@ import base.boudicca.model.structured.StructuredEntry
 import base.boudicca.model.structured.filterKeys
 import base.boudicca.model.structured.toFlatEntry
 import base.boudicca.model.toStructuredEntry
-import base.boudicca.query.AfterExpression
-import base.boudicca.query.AndExpression
-import base.boudicca.query.BeforeExpression
-import base.boudicca.query.ContainsExpression
-import base.boudicca.query.DurationLongerExpression
-import base.boudicca.query.DurationShorterExpression
-import base.boudicca.query.EqualsExpression
-import base.boudicca.query.Expression
-import base.boudicca.query.HasFieldExpression
-import base.boudicca.query.NotExpression
-import base.boudicca.query.OrExpression
-import base.boudicca.query.QueryException
-import base.boudicca.query.Utils
+import base.boudicca.query.*
 import base.boudicca.query.evaluator.util.EvaluatorUtil
 import java.time.LocalDate
 import java.time.OffsetDateTime
 import java.time.ZoneId
 import java.time.format.DateTimeParseException
 import java.util.concurrent.ConcurrentHashMap
 
+@Suppress("detekt:LongMethod", "detekt:CyclomaticComplexMethod")
```

Comment:
maybe we should also refactor this?

Reply:
again, this is not a very complex method, it is just long because there is a long when in there. if you have any better suggestions tell me